code/game/objects/items/stacks/tiles/tile_iron.dm

```diff
@@ -55,6 +55,10 @@
 /obj/item/stack/tile/iron/base //this subtype should be used for most stuff
 	merge_type = /obj/item/stack/tile/iron/base
 
+/obj/item/stack/tile/iron/base/cyborg //cant reskin these, fucks with borg code
```

Comment:
this won't keep borgs from being able to click on pried up floor tiles on the ground with their floor tile module to convert them into metal for their metal synthesizer, right?

Reply:
idk it sohuldnt